src/main/java/org/candlepin/model/ContentOverride.java

```diff
@@ -65,7 +65,7 @@
     @NotNull
     private String name;
 
-    @Size(max = 255)
+    @Size(max = 2048)
```

Comment:
Since the table behind this class is shared with `ActivationKeyContentOverrides`, the annotation should also be updated in that class as well.